Makefile.common

```diff
@@ -105,7 +105,7 @@ BYTERUN:=$(addprefix kernel/byterun/, \
 CORECMA:=lib/clib.cma lib/lib.cma kernel/kernel.cma intf/intf.cma library/library.cma \
         engine/engine.cma pretyping/pretyping.cma interp/interp.cma proofs/proofs.cma \
         parsing/parsing.cma printing/printing.cma tactics/tactics.cma vernac/vernac.cma \
-	stm/stm.cma toplevel/toplevel.cma parsing/highparsing.cma API/API.cma
+	parsing/highparsing.cma stm/stm.cma toplevel/toplevel.cma API/API.cma
```

Comment:
`stm` came before `highparsing` before, so this change makes the reversal necessary?